VKappProject/VKappProject/Controllers/FriendCollectionViewController.swift

```diff
@@ -12,6 +13,7 @@ final class FriendCollectionViewController: UICollectionViewController {
     // MARK: - Private Properties
 
     private var networkService: NetworkServiceProtocol = NetworkService()
```

Comment:
```suggestion
    private let networkService = NetworkService()
```
 раз ты сразу создаешь экземпляр